### PR TITLE
Disable not-fully working wallets

### PR DESCRIPTION
**Pull request by anxolin**
Disables all wallets except metamask.

The idea is to re-enabled them individually when they are better tested.

* WALLET_CONNECT: disabled for now as agreed
* WALLET_LINK: same reason as uni. We need to adapt it. This draft is done, but is untested https://github.com/gnosis/gp-swap-ui/pull/429
* COINBASE_LINK: Opens uniswap right now. It has a code, and I haven't find where can we generate our own. https://go.cb-w.com/mtUDhEZPy1 
* FORTMATIC: Integrated, but we still need to polish something. We need to set the keys per environment.
* Portis: Integrated, but we still need to polish something. We need to set the keys per environment.

### CODE REVIEW

File: src/custom/constants/index.ts [hunk 1]
```diff
@@ -16,6 +17,16 @@ export * from '@src/constants/index'
 
 export const PRODUCTION_URL = 'cow.trade'
 
+const DISABLED_WALLETS = /^(?:WALLET_LINK|COINBASE_LINK|WALLET_CONNECT|FORTMATIC|Portis)$/
+
+// Re-export only the supported wallets
+export const SUPPORTED_WALLETS = Object.keys(SUPPORTED_WALLETS_UNISWAP)
+  .filter(key => !DISABLED_WALLETS.test(key))
+  .reduce((acc, key) => {
```
Comment (W3stside) [inline src/custom/constants/index.ts:25]: nit: you can filter in the reduce

**Review by W3stside — APPROVED** [review]
nice

File: src/custom/constants/index.ts (hunk 1 above)
Comment (anxolin) [reply src/custom/constants/index.ts:25]: ok, i'll add that

File: src/custom/constants/index.ts [hunk 2]
```diff
@@ -16,6 +17,16 @@ export * from '@src/constants/index'
 
 export const PRODUCTION_URL = 'cow.trade'
 
+const DISABLED_WALLETS = /^(?:WALLET_LINK|COINBASE_LINK|WALLET_CONNECT|FORTMATIC|Portis)$/
```
Comment (alfetopito) [inline src/custom/constants/index.ts:20]: I would make this case insentive, just in case

**Review by alfetopito — COMMENTED** [review]
Soft approval

File: src/custom/constants/index.ts (hunk 2 above)
Comment (anxolin) [reply src/custom/constants/index.ts:20]: done

**Review by alfetopito — APPROVED** [review]
(it was meant to be approved already, just didn't selected it so you could address the open points before mergify did it's thing and you needed to open a new PR, etc, etc)

**Pull request merged**